docs/cql/service-levels.rst

```diff
@@ -0,0 +1,29 @@
+==============
+Service Levels
+==============
+
+Service Levels CQL commands
+===========================
+
+``LIST EFFECTIVE SERVICE LEVELS OF <role>``
+-----------------------------------------------------------
+
+Actual values of service level's options may come from different service levels, not only from the one user is assigned with. This can be achived by assigning one role to another.
+
+For instance:
+There are 2 roles: role1 and role2. Role1 is assigned with sl1 (timeout = 2s, workload_type = interactive) and role2 is assigned with sl2 (timeout = 10s, workload_type = batch).
+Then, if we grant role1 to role2, the user with role2 will have 2s timeout (from sl1) and batch workload type (from sl2).
```

Comment:
This statement was completely mysterious to me when I first read it -  why did role2 inherit sl1's timeout and not sl2's? It's because it gets the lower of the two timeouts. This is mentioned in docs/dev/service-levels.rst but apparently missing in this document. Maybe you can just say in parentheses.

Reply:
added that information in parentheses